modules/README.md

```diff
@@ -0,0 +1,24 @@
+[![Crate][crate-image]][crate-link]
+[![Docs][docs-image]][docs-link]
+
+See the [ibc-rs] repo root for more detailed information on this crate can be used.
```

Comment:
It's a bit weird as this file itself is in the repo root and clicking on that link will land on this very same readme.

Reply:
that should point to the README that is one level up. This README is in the modules folder.